### PR TITLE
mvnw files update

### PR DESCRIPTION
**Pull request by MatthewBerk**
**What**:
Updated mvnw files to utilize maven wrapper 0.5.6 instead of 0.5.5

**Why**:
https://github.com/codecentric/chaos-monkey-spring-boot/issues/246#issuecomment-885985299

**How**:
Executing mvn -N io.takari:maven:wrapper

**Checklist**:

- [ ] Documentation added  N/A
- [x] Changelog updated
- [ ] Tests added  N/A
- [x] Ready to be merged

### CODE REVIEW

File: chaos-monkey-docs/src/main/asciidoc/changes.adoc [hunk 1]
```diff
@@ -4,14 +4,13 @@
 Built with Spring Boot {spring-boot-version}
 
 === Bug Fixes
-- https://github.com/codecentric/chaos-monkey-spring-boot/pull/251[#251] Fix missing AssaultPropertiesUpdate validation.
```
Comment (WtfJoke) [inline chaos-monkey-docs/src/main/asciidoc/changes.adoc:7]: These line should stay here

File: chaos-monkey-docs/src/main/asciidoc/changes.adoc [hunk 2]
```diff
@@ -4,14 +4,13 @@
...
+- https://github.com/codecentric/chaos-monkey-spring-boot/pull/252[#252] mvnw files update.
 // - https://github.com/codecentric/chaos-monkey-spring-boot/pull/xxx[#xxx] Added example entry. Please don't remove.
 
 === Improvements
 // - https://github.com/codecentric/chaos-monkey-spring-boot/pull/xxx[#xxx] Added example entry. Please don't remove.
 
 === New Features
- - https://github.com/codecentric/chaos-monkey-spring-boot/pull/241[#239] Added CPU Assault
```
Comment (WtfJoke) [inline chaos-monkey-docs/src/main/asciidoc/changes.adoc:14]: This line as well :)

File: chaos-monkey-docs/src/main/asciidoc/changes.adoc [hunk 3]
```diff
@@ -4,14 +4,13 @@
 Built with Spring Boot {spring-boot-version}
 
 === Bug Fixes
-- https://github.com/codecentric/chaos-monkey-spring-boot/pull/251[#251] Fix missing AssaultPropertiesUpdate validation.
+- https://github.com/codecentric/chaos-monkey-spring-boot/pull/252[#252] mvnw files update.
```
Comment (WtfJoke) [inline chaos-monkey-docs/src/main/asciidoc/changes.adoc:7]: Can you rephrase this into:  "Update Maven Wrapper from 0.5.5 to 0.5.6"

**Review by WtfJoke — COMMENTED** [review]
Thanks for opening this @MatthewBerk. The changes looks as expected, expect the changelog needs some improvements (see my comments) :)

As a tip for future. Always create a separate branch for your contributions. So you can keep your main branch up to date with upstream (either through git tools or github cli, see screenshot)

grafik

File: chaos-monkey-docs/src/main/asciidoc/changes.adoc (hunk 3 above)
Comment (F43nd1r) [reply chaos-monkey-docs/src/main/asciidoc/changes.adoc:7]: As this is a build-only change, should it be in the changelog at all?

Comment (WtfJoke) [reply chaos-monkey-docs/src/main/asciidoc/changes.adoc:7]: It doesnt need to be, but it could. In the past we usually always added an entry (at least for the outside collaborators).

File: chaos-monkey-docs/src/main/asciidoc/changes.adoc (hunk 1 above)
Comment (MatthewBerk) [reply chaos-monkey-docs/src/main/asciidoc/changes.adoc:7]: > Always create a separate branch for your contributions. So you can keep your main branch up to date with upstream/original repository (either through git tools or github ui, see screenshot)

Yeah, I realized that after I did the pull request already. I initially thought it would be fine since I needed to do this to run the project on my system; though realized I couldn't update the master branch with new commits until this pull request closed. 😅

**Review by F43nd1r — APPROVED** [review]
